packages/directory/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "navi-directory",
+  "version": "0.0.0",
+  "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "repository": "",
+  "scripts": {
+    "build": "ember build",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
+    "test": "export COVERAGE=true; ember test --test-port 0"
+  },
+  "dependencies": {
+    "ember-cli-babel": "^6.6.0"
+  },
+  "devDependencies": {
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-ajax": "^3.0.0",
+    "ember-cli": "~2.18.1",
```

Comment:
Do we want to use the latest ember?

Reply:
We do, but we still want it to be compatible with the other repositories. We don't know when the other upgrades would happen.
I will keep it at 2.18 for now, and then do the upgrade later with the others.